<launch>

  <include file="$(find inhus_navigation)/launch/other_launch/_locate_odom.launch"/>
  <node name="rqt_robot_steering" type="rqt_robot_steering" pkg="rqt_robot_steering" output="screen">
    <remap from="/cmd_vel" to="/stage_agents/pr2/cmd_vel"/>
  </node>

</launch>
